README.md

```diff
@@ -87,13 +107,12 @@ If you only plan to use [Unleash client-side React SDK](https://github.com/Unlea
 
 With same access as in the client-side example above you can resolve Unleash feature flags when building static pages.
 
-Use `getFrontendFlags` to load
-
 ```tsx
 import {
   flagsClient,
   getDefinitions,
-  getFrontendFlags,
+  evaluateFlags,
+  // getFrontendFlags,
```

Comment:
Can we update the comment in the code from 

`/* Or with Proxy/Frontend API */` to something like this
`/* Or with the proxy/front-end API (remember to uncomment the getFrontendFlags import) */`?

Reply:
I'll leave it uncommented. Unused variable isn't a big deal when copy-pasting examples.